packages/appium/lib/cli/extension-command.js

```diff
@@ -729,6 +729,24 @@ class ExtensionCliCommand {
       );
     }
 
+    if (!scriptName) {
+      const allScripts = _.toPairs(extScripts);
+      const root = this.config.getInstallPath(installSpec);
+      const existingScripts = await B.filter(
+        allScripts,
+        async ([, p]) => await fs.exists(path.join(root, p))
```

Comment:
do we really need to double check that extensions are telling the truth about script paths? can we not just report which keys are in the scripts object in package json?

Reply:
I don't have any special preferences about that, also not sure why scripts are duplicated in the manifest
For example, doctor script paths I read from the package.json

Reply:
Sure. I guess it doesn't hurt. Wonder if path existence validation might be better done in manifest validation, then we can just rely on the keys being correct. But we can do that another time if you want to just merge this.